gci/views.py

```diff
@@ -14,6 +15,11 @@ def index(request):
     org_id = linked_students[0]['organization_id']
     org_name = linked_students[0]['organization_name']
     s = []
+    
+    get_logo()
+    
+    s.append('<link rel="shortcut icon" type="image/png" href="images/favicon.png"/>')
+    s.append('<image href="images/favicon.png">')
```

Comment:
What's this for? Seems like it's not needed

Reply:
@dob9601 Line 22

Reply:
A review is usually given to the bottommost line in the "review window".

Reply:
my mistake. Line 22 displays the image. Line 21 is a link to the favicon since it is in a png format and in a subfolder

Reply:
Did you mean [`<img>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img) ?

Reply:
can’t believe I did that :( must have been half asleep when working on that